Sofascore/Sofascore/Networking/BaseApiClient.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+protocol BaseApiClient {
+    
+    var baseUrl: String { get }
+    func makeEventUrl(withID eventID: Int) -> URL?
+}
+
+//MARK: Base Implementation
+extension BaseApiClient {
+    
+    var baseUrl: String {
+        return "https://static-api.sofascore.dev/api"
```

Comment:
```suggestion
       "https://static-api.sofascore.dev/api"
```

Jel moze ovako? Mis da bi moglo ic